elasticsearch-plugins/README.rst

```diff
@@ -1,168 +1,191 @@
-      "es.type": "employee",
-      "query": "?q=*",
-      "schema": "{
-        \"type\":\"record\",
-        \"name\":\"etlSchemaBody\",
-        \"fields\":[
-          {\"name\":\"id\",\"type\":\"long\"},
-          {\"name\":\"name\",\"type\":\"string\"},
-          {\"name\":\"age\",\"type\":\"int\"}]}"
-    }
-  }
-
-This example connects to Elasticsearch, which is running locally, and reads in records in the
-specified index (*megacorp*) and type (*employee*) which match the query to (in this case) select all records.
-All data from the index will be read on each run.
-
-Sinks: Batch: Elasticsearch
-===========================
-
-Description
------------
-Batch sink to use Elasticsearch as a sink.
-
-Use Case
---------
-This sink is used whenever you need to write data into Elasticsearch.
-For example, you may want to parse a file and read its contents into Elasticsearch,
-which you can achieve with a stream batch source and Elasticsearch as a sink.
-
-Properties
-----------
-**es.host:** The hostname and port for the Elasticsearch server.
-
-**es.index:** The name of the index where the data will be stored.
-If the index does not already exist, it will be created using
-Elasticsearch's default properties.
-
-**es.type:** The name of the type where the data will be stored.
-If it does not already exist, it will be created.
-
-**es.idField:** The field that will determine the id for the document.
-It should match a fieldname in the structured record of the input.
-
-Example
--------
-::
-
-  {
-   "name": "Elasticsearch",
-      "properties": {
-        "es.host": "localhost:9200",
-        "es.index": "megacorp",
-        "es.type": "employee",
-        "es.idField": "id"
+===============================================
+Elasticsearch Source and Sink Plugin Collection
+===============================================
+
+This project is a collection of Elasticsearch source and sink plugins. These plugins are currently available:
+
+- Elasticsearch Batch Source
+- Elasticsearch Batch Sink
+- Elasticsearch Real-time Sink
+
+Getting Started
+===============
+
+Following are instructions to build and deploy the Hydrator Elasticsearch plugins.
+
+Prerequisites
+-------------
+
+To use the plugins, you must have CDAP version 3.2.0 or later. You can download CDAP Standalone that includes Hydrator `here <http://cask.co/downloads>`__
+
+Build Plugins
+-------------
+
+You get started with Hydrator plugins by building directly from the latest source code::
+
+  git clone https://github.com/caskdata/hydrator-plugins.git
+  cd hydrator-plugins
+  mvn clean package -pl elasticsearch-plugins
+
+After the build completes, you will have a jar for each plugin under the
+``elasticsearch-plugins/target/`` directory.
+
+Deploy Plugins
+--------------
+
+You can deploy the plugins using the CDAP CLI::
+
+  > load artifact target/elasticsearch-plugins-1.1.0-SNAPSHOT-batch.jar \
+         config-file resources/plugin/elasticsearch-batch-plugins.json
```

Comment:
this was recently changed (missed updating the docs).  should be:
`
load artifact target/elasticsearch-plugins-1.1.0-SNAPSHOT.jar \
      config-file target/elasticsearch-plugins-1.1.0-SNAPSHOT.json
`